src/app/service/poap/POAPService.ts

```diff
@@ -11,6 +11,7 @@ import { storePresentMembers } from './start/StartPOAP';
 import { POAPUnclaimedParticipants } from '../../types/poap/POAPUnclaimedParticipants';
 import { POAPTwitterUnclaimedParticipants } from '../../types/poap/POAPTwitterUnclaimedParticipants';
 import ValidationError from '../../errors/ValidationError';
+import cron, { ScheduledTask } from 'node-cron';
```

Comment:
This reminds me, we probably have a series of dependencies we can remove since DEGEN isn't running all the features it used to. Will revisit this in the Garage.